core/tests/protractor.conf.js

```diff
@@ -338,6 +339,27 @@ exports.config = {
       }));
     }
 
+    // Adding a video reporter. For more information see
+    // https://www.npmjs.com/package/protractor-video-reporter
+
+    var ffmpegArguments = [
+      '-y',
+      '-r', '30',
+      '-f', 'x11grab',
+      '-s', '1366x768',
+      '-i', process.env.DISPLAY,
+      '-g', '300',
+      '-vcodec', 'qtrle',
+    ]
+
+    jasmine.getEnv().addReporter(new VideoReporter({
+      baseDirectory: path.resolve(__dirname, '../../../protractor-video'),
```

Comment:
`/protractor-screenshots` is it going to be in the same dir?? (On line 330 I see a different path soo not sure whether they are going to be in the same path?)

Reply:
I think they will be the same path. `__dirname` should be `/home/work/oppia/oppia/core/tests`, so this path is `/home/work/oppia/oppia/core/tests/../../../protractor-video`, which resolves to `/home/work/oppia/protractor-video`. Note that the path on line 330 is relative to the CWD, while this path is relative to the folder enclosing `protractor.conf.js`